notes/change-report-form-received.vbs

```diff
@@ -141,4 +163,50 @@ IF tikl_nav_check = 1 THEN
 	CALL create_MAXIS_friendly_date(date, 10, 5, 18)
 	EMSetCursor 9, 3
```

Comment:
Lines 134: I know this isn't your code, but since this is mandated in the dialog do...loop then the if...statement isn't needed. Please update.

Reply:
Removed unnecessary if...statement in line 134